### PR TITLE
CVE Reporting: Adding parameters used to generate id

### PR DESCRIPTION
**Pull request by jakub-pomykala**
- Adding in parameters used to generate hashed ID
- Moving `getServerHostName()` to KernelUtils class

### CODE REVIEW

**Comment by jakub-pomykala** [comment]
#build

**Comment by jakub-pomykala** [comment]
#libby

**Review by abutch3r — DISMISSED** [review]

**Review by hlhoots — DISMISSED** [review]
I think this look good to me pending a successful build.

**Comment by jakub-pomykala** [comment]
#build

**Comment by jakub-pomykala** [comment]
#libby

**Comment by jakub-pomykala** [comment]
#build

File: dev/io.openliberty.reporting.internal/resources/io/openliberty/reporting/internal/resources/CVEReporting.nlsprops [hunk 1]
```diff
@@ -51,4 +51,8 @@ CWWKF1705.failed.response.useraction=Fix the communication issue and try again.
 
 CWWKF1706.issue.parsing=CWWKF1706W: Issue when parsing Product Information.
 CWWKF1706.issue.parsing.explanation=The service cannot successfully parse the product information.
-CWWKF1706.issue.parsing.useraction=Fix the communication issue and try again.
\ No newline at end of file
+CWWKF1706.issue.parsing.useraction=Fix the communication issue and try again.
+
+CWWKF1707.internal.error=CWWKF1706W: An internal error occurred while checking for security vulnerabilities. The error is: {0}
```
Comment (dmuelle) [inline dev/io.openliberty.reporting.internal/resources/io/openliberty/reporting/internal/resources/CVEReporting.nlsprops]:
```suggestion
CWWKF1707.internal.error=CWWKF1706W: An internal error occurred when the system checked for security vulnerabilities. The error is: {0}
```

Comment (dmuelle) [reply dev/io.openliberty.reporting.internal/resources/io/openliberty/reporting/internal/resources/CVEReporting.nlsprops]: not sure if "system" is the correct term here, but we should avoid elliptical clause "while checking", which can cause translation problems. Also, better state _what_ is doing the checking if possible.

File: dev/io.openliberty.reporting.internal/resources/io/openliberty/reporting/internal/resources/CVEReporting.nlsprops [hunk 2]
```diff
@@ -51,4 +51,8 @@ CWWKF1705.failed.response.useraction=Fix the communication issue and try again.
...
+CWWKF1707.internal.error.explanation=An error occurred that prevented checking for security vulnerabilities.
```
Comment (dmuelle) [inline dev/io.openliberty.reporting.internal/resources/io/openliberty/reporting/internal/resources/CVEReporting.nlsprops]:
```suggestion
CWWKF1707.internal.error.explanation=An error occurred that prevented the system from checking for security vulnerabilities.
```

**Review by dmuelle — COMMENTED** [review]
ID review- let me know if any questions, thanks!

File: dev/io.openliberty.reporting.internal/resources/io/openliberty/reporting/internal/resources/CVEReporting.nlsprops (hunk 1 above)
Comment (jakub-pomykala) [reply dev/io.openliberty.reporting.internal/resources/io/openliberty/reporting/internal/resources/CVEReporting.nlsprops]: Updated to: ```An internal error occurred when the Liberty runtime checked for security vulnerabilities. The error is: {0}```

File: dev/io.openliberty.reporting.internal/resources/io/openliberty/reporting/internal/resources/CVEReporting.nlsprops (hunk 2 above)
Comment (jakub-pomykala) [reply dev/io.openliberty.reporting.internal/resources/io/openliberty/reporting/internal/resources/CVEReporting.nlsprops]: Updated to ```An error occurred that prevented the Liberty runtime from checking for security vulnerabilities.```

File: dev/io.openliberty.reporting.internal/resources/io/openliberty/reporting/internal/resources/CVEReporting.nlsprops [hunk 3]
```diff
@@ -51,4 +51,8 @@ CWWKF1705.failed.response.useraction=Fix the communication issue and try again.
 
 CWWKF1706.issue.parsing=CWWKF1706W: Issue when parsing Product Information.
 CWWKF1706.issue.parsing.explanation=The service cannot successfully parse the product information.
-CWWKF1706.issue.parsing.useraction=Fix the communication issue and try again.
\ No newline at end of file
+CWWKF1706.issue.parsing.useraction=Fix the communication issue and try again.
+
+CWWKF1707.internal.error=CWWKF1706W: An internal error occurred when the Liberty runtime checked for security vulnerabilities. The error is: {0}
+CWWKF1707.internal.error.explanation=An error occurred that prevented the Liberty runtime from checking for security vulnerabilities.
+CWWKF1707.internal.error.useraction=Review the server message.log file and FFDC logs to identify the problem.
```
Comment (dmuelle) [inline dev/io.openliberty.reporting.internal/resources/io/openliberty/reporting/internal/resources/CVEReporting.nlsprops:58]:
```suggestion
CWWKF1707.internal.error.useraction=Review the runtime message.log file and FFDC logs to identify the problem.
```

**Review by dmuelle — APPROVED** [review]
ID reviewed

**Review by hlhoots — APPROVED** [review]
Looks good to me.

**Comment by jakub-pomykala** [comment]
#libby

**Pull request merged**